frontend/src/components/common/GeneralDeleteCamperModal.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Button,
+  HStack,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+
+interface GeneralDeleteCamperModalProps {
+  title: string;
+  bodyText: string;
+  bodyNote?: string;
+  buttonLabel: string;
+  isOpen: boolean;
+  onClose: () => void;
+  onDelete: () => void;
+}
+const GeneralDeleteCamperModal = ({
```

Comment:
I was assuming that this would be used not just for waitlisted campers but normal campers as well - basically general just refers to the face it's not exclusive to one type of camper

Reply:
Makes sense, think we can just call that `DeleteCamperModal`